config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  resources :memberships
+  root "groups#index"
```

Comment:
Small nitpick, but I would reorder this so the root is listed first.

```suggestion
  root "groups#index"
  resources :memberships
```

That way, no matter how many routes you add in the future it is immediately clear what the root route is.